disnake/flags.py

```diff
@@ -1598,6 +1601,45 @@ def automod(self):
         """
         return (1 << 20) | (1 << 21)
 
+    @alias_flag_value
+    def polls(self):
+        """:class:`bool`: Whether guild and direct message polls related events are enabled.
+
+        This is a shortcut to set or get both :attr:`guild_polls` and :attr:`dm_polls`.
+
+        This corresponds to the following events:
+
+        - :func:`on_poll_vote_add` (both guilds and DMs)
+        - :func:`on_poll_vote_remove` (both guilds and DMs)
+        """
+        return (1 << 24) | (1 << 25)
+
+    @flag_value
+    def guild_polls(self):
```

Comment:
The three intents could also use a note similar to the `*_reactions` intents, regarding the cache dependency of `PollAnswer.count`:
https://github.com/DisnakeDev/disnake/blob/cebfb89fd6c9a1711a672b405c7337e5f838bf41/disnake/flags.py#L1438-L1440